Sources/BraintreeCard/CreditCardPOSTBody.swift

```diff
@@ -0,0 +1,134 @@
+// swiftlint:disable all
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+struct CreditCardPOSTBody: Encodable {
+    var authenticationInsight: Bool?
+    var merchantAccountID: String?
+    var meta: Meta?
+    let creditCard: CreditCard?
+    
+    enum CodingKeys: String, CodingKey {
+        case authenticationInsight
+        case meta = "_meta"
+        case merchantAccountID = "merchantAccountId"
+        case creditCard = "credit_card"
+    }
+
+    init(
+        card: BTCard,
+        metaData: BTClientMetadata
+    ) {
+        self.creditCard = CreditCard(card: card)
+        
+        if card.authenticationInsightRequested {
+            self.authenticationInsight = card.authenticationInsightRequested
+            self.merchantAccountID = card.merchantAccountID
+        }
+        
+        self.meta = Meta(metaData: metaData)
+    }
+
+    struct Meta: Encodable {
+        var integration: String
+        var source: String
+        var sessionId: String
```

Comment:
nit to match the casing we use in the rest of the SDK:
```suggestion
        var sessionID: String
```